backend/corpora/lambdas/api/v1/collection_uuid/publish.py

```diff
@@ -24,8 +24,8 @@ def post(collection_uuid: str, body: object, user: str):
         raise ForbiddenHTTPException()
     if all([dataset.tombstone for dataset in collection.datasets]):
         raise ConflictException(detail="The collection must have a least one dataset.")
-
+    collection_id = collection.revision_of if collection.revision_of else collection.id
```

Comment:
We are reusing this logic in a number of places. We should move it into its own function to simplify our lives and reduce the cognitive load on our brains.

Reply:
Concern noted but I only count two ✌️ at the moment